frontend-opal/src/components/WatchList/InstrumentItems/instrumentItems.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+
+const InstrumentItems = ({ infos }) => (
```

Comment:
@martinupset instead of infos call it `instruments`

Reply:
ok